superset-frontend/src/views/CRUD/data/database/state.ts

```diff
@@ -21,5 +21,5 @@ import { useSelector } from 'react-redux';
 import { ViewState } from 'src/views/types';
 
 export function useCommonConf() {
-  return useSelector((state: ViewState) => state.common.conf);
+  return useSelector((state: ViewState) => state?.common?.conf);
```

Comment:
why did we need to add `?` here? seems like it worked fine before?

Reply:
it will break unit test `AlertReportModal.test.jsx`: it didn't read conf before this PR.